Scripts/Data/Collection/GameObjectObservableSet.cs

```diff
@@ -3,6 +3,7 @@
     using UnityEngine;
     using UnityEngine.Events;
     using System;
+    using System.Collections.Generic;
```

Comment:
why has this been added? if it was needed wouldn't we have had errors before?

Reply:
Because of the XML documentation comments: If a type is `cref`'ed in any comment it actually needs to be "in the scope" for IntelliSense to properly pick that up.